Cargo.lock

```diff
@@ -144,9 +144,9 @@ checksum = "a1fd36ffbb1fb7c834eac128ea8d0e310c5aeb635548f9d58861e1308d46e71c"
 
 [[package]]
 name = "arbitrary"
```

Comment:
it's test only (libfuzz) + we don't use it. I'm wondering how it got updated? What are the attack vectors when a backdoor gets inserted in a transitive dependency that we don't use? For example, if we type `cargo build` will this run the `build.rs` of this dependency?

Reply:
> if we type cargo build will this run the build.rs of this dependency?
If that dep is apart of the dep graph, then yes it will be run